ddtrace/llmobs/_llmobs.py

```diff
@@ -80,18 +79,11 @@ def __init__(self, tracer=None):
         )
 
     def _configure_span_writer(self):
-        if config._llmobs_agentless_enabled:
-            self._llmobs_span_writer = LLMObsSpanWriter(
-                site=config._dd_site,
-                api_key=config._dd_api_key,
-                interval=float(os.getenv("_DD_LLMOBS_WRITER_INTERVAL", 1.0)),
-                timeout=float(os.getenv("_DD_LLMOBS_WRITER_TIMEOUT", 5.0)),
-            )
-        else:
-            self._llmobs_span_writer = LLMObsSpanAgentWriter(
-                interval=float(os.getenv("_DD_LLMOBS_WRITER_INTERVAL", 1.0)),
-                timeout=float(os.getenv("_DD_LLMOBS_WRITER_TIMEOUT", 5.0)),
-            )
+        self._llmobs_span_writer = LLMObsSpanWriter(
+            is_agentless=config._llmobs_agentless_enabled,
+            interval=float(os.getenv("_DD_LLMOBS_WRITER_INTERVAL", 1.0)),
+            timeout=float(os.getenv("_DD_LLMOBS_WRITER_TIMEOUT", 5.0)),
+        )
```

Comment:
I feel like this doesn't need its own function anymore, can we just directly start the `LLMObsSpanWriter` in `LLMObs.__init__()`?